roles/test/robocup.role

```diff
@@ -0,0 +1,69 @@
+# This role is used for running the full autonomous soccer set up with a real robot. It should be used with a computer
```

Comment:
whats the difference between this and robocup?

Reply:
Robocup doesn't have Networking stuff which we always need to add when testing, maybe we could just have all of that in the main robocup